app/util/data_preparation/confluence/prepare-data.py

```diff
@@ -90,6 +90,13 @@ def write_test_data_to_files(dataset):
     __write_to_file(CONFLUENCE_USERS, users)
 
 
+def __check_collaborative_editing_enable(confluence_api):
```

Comment:
For consistency let's rename method name to `__is_collaborative_editing_enabled`

Reply:
Done